portfolio/src/main/java/com/google/sps/servlets/CheckLoginServlet.java

```diff
@@ -31,26 +31,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           UserService userService = UserServiceFactory.getUserService();
           boolean isloggedin = false;
           String url = "/";
+          String email = "null";
```

Comment:
Probably better to have this be an empty string rather than a string literal of "null"